plugins/module_utils/ec2.py

```diff
@@ -59,6 +58,9 @@
 from .policy import compare_policies  # pylint: disable=unused-import
 from .policy import sort_json_policy_dict  # pylint: disable=unused-import
 
+# Used to live here, moved into # ansible_collections.amazon.aws.plugins.module_utils.retries
+from .retries import AWSRetry  # pylint: disable=unused-import
+
 BOTO_IMP_ERR = None
 try:
     import boto
```

Comment:
Do we need to remove this as well?

Reply:
The "import boto"?  That should go away with #630.

I've been trying to split up some of the changes I'd like to make.  I've been having "fun" with circular dependencies due to where certain things originally lived.

Reply:
Yes, exactly! Ok, thanks! LGTM!